shared/fs/index.js

```diff
@@ -17,10 +17,16 @@ type FolderProps = {
   path: Types.Path,
   resetParticipants: Array<string>,
   routePath: I.List<string>,
+  onAttach: (path: Types.Path, local: Array<string>) => void,
 }
 
-class Files extends React.PureComponent<FolderProps> {
+class Files extends React.PureComponent<FolderProps, State> {
+  //      this.props.onAttach(this.props.path, paths)
+
   render() {
+    // TODO check whether the path is writable by the user.
```

Comment:
[Here](https://github.com/keybase/client/blob/1f621f5e35f0e77c49c0d507667e7b02c2ababbf/shared/fs/header/add-new-container.js#L26)'s how we did it in the "Add New" button.

Reply:
Also since this would be done in the container, it seems we could also just make `onAttach` a `null` if it shouldn't be enabled.